jobs/integration/templates/microbot.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
```

Comment:
lifted this file straight from the reactive archive of the kubernetes-worker charm
https://github.com/charmed-kubernetes/charm-kubernetes-worker/blob/reactive-archive/templates/microbot-example.yaml

Reply:
rather than adding it back to the kubernetes-worker charm -- just use this sample to start the microbot in tests